.github/actions/bazel-test-all/action.yaml

```diff
@@ -51,6 +51,9 @@ runs:
 
       - name: Run Bazel Commands
         uses: ./.github/actions/bazel
+        env:
+          MERGE_BASE_SHA: ${{ github.event.pull_request.base.sha }}
+          RANCH_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
```

Comment:
Should be `BRANCH_HEAD_SHA` 😅

Should the `diff.sh` script be updated to fail if these are unset? Right now they default to "`HEAD`" and "", if not.

Reply:
Thanks fixed!